Userland/Libraries/LibUnicode/Locale.cpp

```diff
@@ -834,6 +834,15 @@ Optional<StringView> resolve_subdivision_alias(StringView subdivision)
 #endif
 }
 
+Optional<LanguageID> add_likely_subtags([[maybe_unused]] LanguageID const& language_id)
+{
+#if ENABLE_UNICODE_DATA
```

Comment:
`if constexpr` this?

Reply:
IIRC we can't use `if constexpr` here because not all builds hit the cmake path that set `ENABLE_UNICODE_DATA` to 0 or 1. I think it's the fuzzer build. I can make it define this thing, but I'd do that in a separate change since it'd touch functions here and in other LibUnicode files

Reply:
Ah, then let's just keep `#if`, I think other places use `#if` too.